### DISCUSSION TITLE
有关即食罐头版本更新

### DISCUSSION THREAD

**Discussion by sugarcanecat** [theme-shoka-x/hexo-theme-shokaX#366] [Q&A] [answered]
1. 请问如果想更新主题到最新版本，该如何操作呢？之前用的是即食罐头，即食罐头版本是0.4.14，目前版本是0.4.16，是只需要修改 shokax-can/package.json 中的"hexo-theme-shokax": "0.4.14"为“0.4.16”，然后在“shokax-can”目录下重新pnpm install就可以更新最新版本主题吗？
2. 我查看我的即食罐头中 “_config.shokax.yml”没有任何内容，如何做到更新最新主题版本呢？
3. 不太明白“npm源”的具体操作是什么？

**Comment by D-Sketon** [comment, marked as answer]
1. 0.4.14-0.4.16 依赖没有什么变化，可以直接pnpm install
2. _config.shokax.yml 的作用是让你在里面编写需要覆盖原 _config.yml 的相关配置，这样当主题更新后你的自定义配置不会丢失
